synapse/models/inet.py

```diff
@@ -1774,6 +1783,9 @@ def getModelDefs(self):
                         'interfaces': ('inet:service:base',),
                         'props': (
 
+                            ('app', ('inet:service:app', {}), {
```

Comment:
Should this be added to `inet:service:base` ?

Reply:
maybe, but i wanted to noodle a bit before we do.

Reply:
One use would be to tie multiple `inet:service:instance` nodes to a common `:app`; allowing for multiple discrete instances of an application?

Reply:
Scratch that. We probably just need an `:app` property on inet:service:instance instead.